src/test/movieSlice.test.js

```diff
@@ -0,0 +1,36 @@
+import moviesSlice, { fetchMovies } from '../data/moviesSlice'
```

Comment:
Consider moving tests to their corresponding util/service/component directory for a better project structure, tests don't have to be declared all in one place...